packages/fxa-settings/src/components/Nav/index.stories.tsx

```diff
@@ -3,33 +3,67 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-import { storiesOf } from '@storybook/react';
-import { mockSession } from '../../models/mocks';
+import { LocationProvider } from '@reach/router';
 import { getDefault } from '../../lib/config';
 import { Nav } from '.';
 import { AppContext } from 'fxa-settings/src/models';
+import { mockAppContext } from '../../models/mocks';
+import { Account } from '../../models/Account';
+import { MOCK_LINKED_ACCOUNTS } from '../LinkedAccounts/mocks';
+
+const defaultStory = { title: 'Components/Nav', component: Nav };
```

Comment:
Updated format per this comment https://github.com/mozilla/fxa/pull/12870#issuecomment-1126134918